src/App.js

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
 const App = () => (
-  <h1>React sum</h1>
+  <>
+    <Sum a={2} b={1} />
```

Comment:
"App component should render 3 Sum components for 2 + 2, 10 + 0 and -5 + 5", here 2 + 1 instead 2 + 2